pyaedt/icepak.py

```diff
@@ -6312,3 +6425,45 @@ def create_square_wave_transient_assignment(self, on_value, initial_time_off, on
             Boundary dictionary object that can be passed to boundary condition assignment functions.
         """
         return SquareWaveDictionary(on_value, initial_time_off, on_time, off_time, off_value)
+
+
+class IcepakDesignSettingsManipulation:
```

Comment:
Would it be possible to define a `DesignSettingsManipulation` interface which `IcepakDesignSettingsManipulation` should implement ? The idea would be to "follow" the strategy design pattern. However, I don't know to what extent this is compatible with IronPython (if these line of codes are concerned ofc)